src/psyclone/tests/domain/lfric/transformations/lfric_extract_test.py

```diff
@@ -90,7 +91,8 @@ def test_node_list_error(tmpdir):
     assert ("Error in LFRicExtractTrans: Argument must be "
             "a single Node in a Schedule, a Schedule or a list of Nodes in a "
             "Schedule but have been passed an object of type: "
-            "<class 'psyclone.dynamo0p3.DynInvoke'>") in str(excinfo.value)
+            "<class 'psyclone.domain.lfric.lfric_invoke.LFRicInvoke'>") \
+           in str(excinfo.value)
```

Comment:
You don't need line continuation `\` if you put the bracket after `in str(excinfo.value)`. Please also align `in` with the quotations above, i.e.
```
            "<class 'psyclone.domain.lfric.lfric_invoke.LFRicInvoke'>"
            in str(excinfo.value))
```